### DISCUSSION TITLE
识别不到本地的中文版

### DISCUSSION THREAD

**Discussion by w2534073922** [k-skye/gitkraken-chinese#9] [General]
不知道怎么识别本地的上个版本翻译
![image](https://github.com/yk47g/gitkraken-chinese/assets/65179645/6b028fdf-5034-4df0-b287-620e5d7baa09)
![image](https://github.com/yk47g/gitkraken-chinese/assets/65179645/cfcd2905-7483-465c-b609-de2d7ccb9596)

**Comment by YuanXiQWQ** [comment]
现在再拉取试试,应该修复了
